src/lib/Behat/PageObject/Page.php

```diff
@@ -69,8 +80,8 @@ public function verifyRoute(): void
      *
      * @return string
      */
-    public function getPageHeaderTitle(): string
+    public function getPageTitle(): string
     {
-        return $this->context->findElement('.ez-page-title')->getText();
+        return $this->context->findElement('.ez-header h1')->getText();
```

Comment:
Just to make sure: have you run studio tests? They were relying on that property. Changing the method name for use breaks it in Studio.